src/dao/pages/ProposalDetailsPage/ProposalDetailsPage.tsx

```diff
@@ -133,7 +133,7 @@ export function ProposalDetailsPage() {
             ) : undefined}
             {isMultiSigProposal && transactionHash ? (
               <ProposalConfirmationDetails
-                safeAccountId={safeAccountId}
+                safeEVMAddress={safeAccountId}
```

Comment:
prop `safeEVMAddress ` value should also be `safeEVMAddress `

Reply:
Done.